spec/features/save_and_add_names_to_crawls_spec.rb

```diff
@@ -17,9 +14,14 @@
         expect(page).to have_content("Name your crawl")
       end
       Then "I can change the crawl name and save my changes" do
+        c1 = Crawl.last
```

Comment:
@cabelhigh is this dependent on the order the specs run? wouldn't it be messed up if other specs were not run before it?